managing-data.html

```diff
@@ -174,15 +335,83 @@
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
+</ol>
+	
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<h2><a class="anchor" id="keyvalue"></a>Key-value pairs in OMERO 5.1</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p>OMERO version 5.1 allows you to annotate Projects, Datasets and Images with key-value pairs. The key-value pairs are indexed by OMERO.server, so are fully searchable.</p>
+
+<p>This functionality is exactly the same for OMERO.insight and OMERO.web.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<ol class="step">
+
+<li><p class="first">The key-value pairs table is located in the Annotations section of the <span class="bold">General</span> pane on the right hand side.</p>
+
+<p>Click the <span class="bold">+</span> button or click on an empty cell in the table to add a key-value pair.</p>
+
+<p>Enter text for the key and use <span class="bold">Return</span> or <span class="bold">Tab** to move to the value field.</p>
```

Comment:
Error in the bold formatting here - you've got the ** left in and bold tag isn't closed until the end of the sentence.